corehq/apps/app_manager/detail_screen.py

```diff
@@ -98,6 +100,10 @@ def __init__(self, app, module, detail, column, sort_element=None,
         self.sort_element = sort_element
         self.order = order
         self.id_strings = id_strings
+        self.parent_tab_nodeset = parent_tab_nodeset
+
+    def add_sort_node_for_nodeset_field(self):
```

Comment:
Minor, but the name here confused me, `add` sounds like this function alters the column. Maybe `should_sort_nodeset_detail_tab` or `has_sort_node_for_nodeset_detail`?